src/main/scala/com/databricks/spark/redshift/RedshiftRelation.scala

```diff
@@ -62,6 +65,8 @@ private[redshift] case class RedshiftRelation(
     }
   }
 
+  override def toString: String = s"RedshiftRelation($tableNameOrSubquery)"
```

Comment:
This change makes explain() output look a little nicer and also helps to prevent credentials from appearing in log messages.